.lagoon.env

```diff
@@ -0,0 +1 @@
+ENABLE_REDIS="true"
```

Comment:
let's not create a new environment variable, but instead remove the `getenv('ENABLE_REDIS')` check in the settings.php

Reply:
Check removed. Fixed